wordpress-mobile/WordPress-Aztec-iOS
# Reorganized the file structure.

Reorganizes the file structure.

Removes the private / public separation, as it was needlessly complicated, and not very helpful in organizing the project.

Tries to provide a cleaner folder structure.

**How to test:**

Make sure unit tests run correctly.
Make sure the app runs correctly, and editing works normally.
